### DISCUSSION TITLE
What tool was used to create the graphics in this article?

### DISCUSSION THREAD

**Discussion by aaron-ai** [open-telemetry/opentelemetry-specification#3308] [Q&A] [answered]
Hello, I have a question that may not be directly related to the specifications. I was wondering what tool was used to create the graphics in [this article](https://github.com/open-telemetry/opentelemetry-specification/blob/main/internal/img/architecture.png)? They are really aesthetically pleasing.

Thank you!

**Comment by trask** [comment]
cc @tedsuo

**Comment by tedsuo** [comment, marked as answer]
I drew it in Procreate :)
